doc/CMakeLists.txt

```diff
@@ -181,30 +181,30 @@ if(BUILD_DOCS AND DOXYGEN_FOUND)
 
   # js interactive tutorials
   set(opencv_tutorial_html_dir "${CMAKE_CURRENT_BINARY_DIR}/doxygen/html")
-  set(js_interactive_tutorials_dir "${CMAKE_CURRENT_SOURCE_DIR}/js_tutorials/js_interactive_tutorials")
+  set(js_tutorials_assets_dir "${CMAKE_CURRENT_SOURCE_DIR}/js_tutorials/js_assets")
   set(ocv_js_dir "${CMAKE_BINARY_DIR}/bin")
   set(ocv_js "opencv.js")
 
-  set(opencv_interactive_tutorials_deps "")
+  set(js_tutorials_assets_deps "")
 
   # make sure the build directory exists
   file(MAKE_DIRECTORY "${opencv_tutorial_html_dir}")
 
   # gather and copy specific files for js test
-  file(GLOB_RECURSE js_interactive_html RELATIVE "${js_interactive_tutorials_dir}" "${js_interactive_tutorials_dir}/*")
-  foreach(f ${js_interactive_html})
+  file(GLOB_RECURSE js_assets RELATIVE "${js_tutorials_assets_dir}" "${js_tutorials_assets_dir}/*")
+  foreach(f ${js_assets})
     add_custom_command(OUTPUT "${opencv_tutorial_html_dir}/${f}"
-                       COMMAND ${CMAKE_COMMAND} -E copy_if_different "${js_interactive_tutorials_dir}/${f}" "${opencv_tutorial_html_dir}/${f}"
-                       DEPENDS "${js_interactive_tutorials_dir}/${f}"
+                       COMMAND ${CMAKE_COMMAND} -E copy_if_different "${js_tutorials_assets_dir}/${f}" "${opencv_tutorial_html_dir}/${f}"
+                       DEPENDS "${js_tutorials_assets_dir}/${f}"
                        COMMENT "Copying ${f}"
                       )
-    list(APPEND opencv_interactive_tutorials_deps "${js_interactive_tutorials_dir}/${f}" "${opencv_tutorial_html_dir}/${f}")
+    list(APPEND js_tutorials_assets_deps "${js_tutorials_assets_dir}/${f}" "${opencv_tutorial_html_dir}/${f}")
   endforeach()
 
   # add dep on opencv.js target
-  list(APPEND opencv_interactive_tutorials_deps ${ocv_js})
+  list(APPEND js_tutorials_assets_deps "${ocv_js_dir}/${ocv_js}")
 
-  add_custom_target(copy_js_interactive_tutorials
+  add_custom_target(copy_js_assets
                     COMMAND ${CMAKE_COMMAND} -E copy_if_different "${ocv_js_dir}/${ocv_js}" "${opencv_tutorial_html_dir}/${ocv_js}"
-                    DEPENDS ${opencv_interactive_tutorials_deps})
+                    DEPENDS ${js_tutorials_assets_deps})
```

Comment:
why depends on assets? it should only depend on `${ocv_js_dir}/${ocv_js}`, correct?